src/envoy/http/authn/authenticator_base.cc

```diff
@@ -70,13 +62,27 @@ void AuthenticatorBase::validateX509(
 }
 
 void AuthenticatorBase::validateJwt(
-    const iaapi::Jwt&,
+    const iaapi::Jwt& jwt,
     const AuthenticatorBase::MethodDoneCallback& done_callback) {
   Payload payload;
   Envoy::Http::HeaderMap& header = *filter_context()->headers();
   ENVOY_LOG(debug, "{} the number of headers is {}", __func__, header.size());
 
-  bool ret = AuthnUtils::GetJWTPayloadFromHeaders(header, kJwtHeaderKey,
+  if (0 ==
+      filter_context()->filterConfig()->jwt_output_payload_locations().count(
+          jwt.issuer())) {
+    ENVOY_LOG(
+        error,
+        "AuthenticatorBase: {}(): no JWT payload header location is found "
+        "for the issuer {}",
+        __func__, jwt.issuer());
```

Comment:
Remove prefix. LOG already adds to file + line to log message.

Reply:
Done.